.github/workflows/ci.yml

```diff
@@ -16,6 +16,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Dump Mailjet token into the .env file
```

Comment:
Instead of dumping into `.env` would recommend just setting the env variables directly (see https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables) - `.env` is normally just for developers locally although that's more just a nice to have.

Otherwise looks good!

Reply:
I got rid of that step altogether! Now the env variable gets passed directly to the step involving docker-compose that needs it.

Reply:
Looks good! Great work :)